source/common/http/filter/ratelimit.cc

```diff
@@ -117,9 +117,16 @@ void Filter::complete(Envoy::RateLimit::LimitStatus status) {
     break;
   case Envoy::RateLimit::LimitStatus::OverLimit:
     cluster_->statsScope().counter("ratelimit.over_limit").inc();
-    Http::CodeUtility::ResponseStatInfo info{config_->scope(), cluster_->statsScope(), EMPTY_STRING,
-                                             *getTooManyRequestsHeader(), true, EMPTY_STRING,
-                                             EMPTY_STRING, EMPTY_STRING, EMPTY_STRING, false};
+    Http::CodeUtility::ResponseStatInfo info{config_->scope(),
```

Comment:
Do we want this style of overflow for initializers? Seems it would make sense to have them match function argument style, where I think we are packing as much as we can into the line.